### ISSUE TITLE
Property 'after' does not exist on type 'ColumnBuilder'

### ISSUE THREAD

**Issue by o8e** [knex/knex#3549]
# Environment

Knex version: ^0.20.2
Database + version: MySQL
OS: macOS 10.14.6

_@lorefnon / typescript_

# Bug

1. Explain what kind of behaviour you are getting and how you think it should do

Should be able to use after() on ColumnBuilder

2. Error message

`Property 'after' does not exist on type 'ColumnBuilder'. Did you mean 'alter'?ts(2551)`

3. Reduced test code, for example in https://npm.runkit.com/knex or if it needs real
   database connection to MySQL or PostgreSQL, then single file example which initializes
   needed data and demonstrates the problem.

```js
knex.schema.alterTable('users', (t: Knex.TableBuilder) => {
      t.integer('creator_id')
        .after('source') // <== No worky
        .unsigned()
      t.foreign('creator_id').references('users.id')
    })
```

**Comment by o8e** [comment]
@lorefnon would really like to remove `//@ts-ignore` from these migrations soon. Any idea?

**Comment by carvalholeo** [comment]
Does someone have any updates? I'd like to use this typed column builder, but I get an error always I use it.

@wwwouter can you look at this?

**Comment by kibertoad** [comment]
@lorefnon

**Comment by wwwouter** [comment]
> Does someone have any updates? I'd like to use this typed column builder, but I get an error always I use it.
> 
> @wwwouter can you look at this?

Yeah, I think @lorefnon is best suited to answer this 😅

**Comment by iamchathu** [comment]
Any update on this?

**Comment by kibertoad** [comment]
PRs are welcome :)

**Comment by kibertoad** [comment]
Released in 0.21.15
